qiskit/aqua/utils/run_circuits.py

```diff
@@ -366,9 +357,9 @@ def run_on_backend(backend, qobj, backend_options=None,
                     config['noise_model'] = temp_noise_config
                 qobj.config = QasmQobjConfig.from_dict(config)
 
-            job = AerJob(backend, job_id, backend._run, qobj)
-            job.submit()
+            job = backend.run(qobj)
```

Comment:
You might want to explicitly add `validate=False` here since the default is set to True for aer 0.7 (this isn't qobj schema validation but Aer specific validation)

```suggestion
            job = backend.run(qobj, validate=False)
```